bugbear.py

```diff
@@ -1164,31 +1163,30 @@ def check_for_b903(self, node):
         self.errors.append(B903(node.lineno, node.col_offset))
 
     def check_for_b018(self, node):
```

Comment:
this is mostly the same - just removing the toplevel for loop and de-indenting, renaming loopvar `subnode` to `node`. Can turn on "ignore-whitespace" option on PR view for easier review